### DISCUSSION TITLE
Can't get custom theme to load

### DISCUSSION THREAD

**Discussion by jengalas** [giscus/giscus#564] [Q&A]
I'm working with a Jekyll site on localhost and trying to create a custom giscus theme based on the information here:  https://github.com/giscus/giscus/blob/main/ADVANCED-USAGE.md#data--attributes

I have the URL specified in the `script` tag like this: `data-theme="http://localhost:4000/assets/css/giscus_theme.css"`

But I get this error: **Access to CSS stylesheet at 'http://localhost:4000/assets/css/giscus_theme.css' from origin 'https://giscus.app' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.**

Any ideas what I'm missing here? Thank you!

**Comment by laymonage** [comment]
Sounds similar to https://github.com/jekyll/jekyll/issues/8889, maybe try accessing the server from `127.0.0.1:4000` instead of `localhost;4000`.

**Reply by jengalas** [reply]
Thanks, but that didn't seem to make a difference. I sent it up to Netlify (example post: https://jocular-tayaki-bd1369.netlify.app/2021/08/20/captivating-math-now-with-python) and I'm getting the same CORS error there.

**Reply by jengalas** [reply]
Update: I have it working now on Netlify. I had to create a configuration file (`netlify.toml`) and configure the Access-Control-Allow-Origin header there. I'm still not having any luck on localhost, though.
